lms/envs/test.py

```diff
@@ -557,3 +557,6 @@
 
 # Financial assistance page
 FEATURES['ENABLE_FINANCIAL_ASSISTANCE_FORM'] = True
+
+# Course Content Bookmarks Settings
+MAX_BOOKMARKS_PER_COURSE = 5
```

Comment:
Instead of overridding it here, we should lower the limit for just the tests which test this. You can use:
```
@patch('django.conf.settings.MAX_BOOKMARKS_PER_COURSE, 5)
```